appveyor.yml

```diff
@@ -14,14 +14,14 @@ environment:
   innounp: '%scoop_helpers%\innounp\innounp.exe'
   matrix:
     - PowerShell: 5
-    - PowerShell: 6
+    - PowerShell: Current
```

Comment:
Can you share an official link which mentions this is the correct behaviour to get latest Powershell?

Reply:
The matrix just mentions the name, this part is not responsible for pulling in PowerShell

Reply:
So would using PowerShell 5 for example lead to it also using PowerShell's newest version? That does sound a bit weird to me.

Not very well versed with AppVeyor.

Reply:
@pratikpc the names here are just matrix keys so the job gets run once for `powershell` and once for `pwsh`, changing the name stops the misconception that we're using PS 6 as AppVeyor keeps the version of `pwsh` current (so v7 ATM).

Reply:
@stevehipwell ps indicates PowerShell 5 while pwsh indicates PowerShell's newest version right?